src/coin_selector.rs

```diff
@@ -419,6 +429,40 @@ impl<'a> CoinSelector<'a> {
         }
     }
 
+    /// The value of the change output should have to drain the excess value while maintaining the
+    /// constraints of `target` and respecting `change_policy`.
+    ///
+    /// If not change output should be added according to policy then it will return `None`.
+    pub fn drain_value(&self, target: Target, change_policy: ChangePolicy) -> Option<u64> {
+        let excess = self.excess(
+            target,
+            Drain {
+                weights: change_policy.drain_weights,
+                value: 0,
+            },
+        );
+        if excess > change_policy.min_value as i64 {
```

Comment:
I'm wondering if this should have been `>=` instead of `>` . The `min_value_and_waste` change policy was previously returning change for excess equal to the min value.

Reply:
I think it should be `>=` indeed.